src/task_8_02.py

```diff
@@ -0,0 +1,18 @@
+""" Даны три слова. Выяснить, является ли хоть одно из них палиндромом
+("перевертышем"), т. е. таким, которое читается одинаково слева направо и
+справа налево. (Определить функцию, позволяющую распознавать слова
+палиндромы.)[03-10.32]
+"""
+list_exp = ['ivavi', 'nana', 'opo']
+
+
+def palindrom_(*args):
```

Comment:
Добавь сделаем так, чтобы функция принимала 1 слово и возвращала либо True либо False
```suggestion
def is_palindrom(*args):
```